docs/src/modules/java-protobuf/pages/actions.adoc

```diff
@@ -360,6 +360,20 @@ action returning and the `addItem` call from the action would lead to more items
 
 Such validation that depends on state can only be done safely while handling the command inside the entity.
 
+== Actions as Life-cycle Hooks
+
+An Action's gRPC method can be triggered automatically when some predefined service life-cycle event happens, serving as a custom hook. For such use, the method needs be annotated with a specific flag as shown below and its must be of type `google.protobuf.Empty`.
```

Comment:
> when some predefined service life-cycle event happens

This gives the impression that we can chose from a list of life-cycle events, but we can only do it on start-up. Better to be explicit about it.

Reply:
yeah, I was preparing for other events being added.. but maybe better to be explicit for now. I will update.